src/tests.c

```diff
@@ -3332,13 +3334,19 @@ void run_wnaf(void) {
     int i;
     secp256k1_scalar n = {{0}};
 
+    test_constant_wnaf(&n, 4);
     /* Sanity check: 1 and 2 are the smallest odd and even numbers and should
      *               have easier-to-diagnose failure modes  */
     n.d[0] = 1;
     test_constant_wnaf(&n, 4);
     n.d[0] = 2;
     test_constant_wnaf(&n, 4);
-    /* Test 0 */
+    /* Test -1, because it's a special case in wnaf_const */
+    n = secp256k1_scalar_one;
+    secp256k1_scalar_negate(&n, &n);
+    test_constant_wnaf(&n, 4);
+
+    /* Test 0 for fixed wnaf */
```

Comment:
This comment should be in above line 3337?

Reply:
There's a difference between constant and "fixed" wnaf. This comment is about the fixed wnaf call below. If that's confusing I think we can just remove it.

Reply:
Okay, I really got confused because the commit message said "This commit also adds a test for 0." and then my impression was that you added the test but the comment in different places. 

Don't remove it. The resulting code is not confusing, just the diff confused me.